### ISSUE TITLE
Did'nt get Address or Buyer info after generating RDT (RestrictedDataToken)

### ISSUE THREAD

**Issue by webinsect-rishabh** [clousale/amazon-sp-api-php#92]
I generated RDT (RestrictedDataToken) to get details like buyer info and address of orders. But after getting RDT token i still getting only that much data which i was getting with normal access token. Do i require any other permissions in my developer profile ? Can anyone help me on this ?

**Comment by adarmanto** [comment]
I got the same issue. Did you resolve this, @webinsect-rishabh?

**Comment by webinsect-rishabh** [comment]
@adarmanto do you still able to generate RDT (RestrictedDataToken) because now i am getting error on createRestrictedDataToken()?

"errors": [
    {
      "code": "InvalidInput",
      "message": "Application do not have access to some or all requ (truncated...)

**Comment by adarmanto** [comment]
@webinsect-rishabh Yep, I do.

Looking at the error message seems that you are no longer have the correct access, I suggested you contact Amazon Support to follow up.

**Comment by webinsect-rishabh** [comment]
Do you got PII permission in your developer profile ?

**Comment by adarmanto** [comment]
> @adarmanto Do you got PII permission in your developer profile ?

Yep, we did after waiting a bit long response by submitting the form, etc :)
